addons/egoventure/nodes/eight_side_room.gd

```diff
@@ -0,0 +1,245 @@
+		$Camera/Left.rect_position.x = 0
+		$Camera/Left.rect_position.y = EgoVenture.configuration.inventory_size
+		$Camera/Left.rect_size.x = EgoVenture.configuration\
+				.tools_navigation_width
+		$Camera/Left.rect_size.y = _viewport_size.y -\
+				EgoVenture.configuration.inventory_size
+		$Camera/Right.rect_position.x = _viewport_size.x -\
+				EgoVenture.configuration.tools_navigation_width
+		$Camera/Right.rect_position.y = EgoVenture.configuration.inventory_size
+		$Camera/Right.rect_size.x = EgoVenture.configuration\
+				.tools_navigation_width
+		$Camera/Right.rect_size.y = _viewport_size.y -\
+				EgoVenture.configuration.inventory_size
+		EgoVenture.connect("requested_view_change", self, "_set_current_view")
+
+
+# Properly position the different views
+# Navigate to the default view when we're not in the editor
+# Also check EgoVenture.target_view wether we need to directly switch
+# to a different view
+func _enter_tree():
+	if not Engine.editor_hint:
+		if EgoVenture.target_view != VIEW_UNSET:
+			_set_current_view(EgoVenture.target_view)
+		else:
+			_set_current_view(default_view)
+	$Views/Front.position = Vector2(0, _viewport_size.y * -1 - TEXTURE_DISTANCE)
+	$Views/Right.position = Vector2(_viewport_size.x + TEXTURE_DISTANCE, 0)
+	$Views/Back.position = Vector2(0, _viewport_size.y + TEXTURE_DISTANCE)
+	$Views/Left.position = Vector2(_viewport_size.x * -1 - TEXTURE_DISTANCE, 0)
+	$Views/FrontLeft.position = Vector2(_viewport_size.x * -1 - TEXTURE_DISTANCE, _viewport_size.y * -1 - TEXTURE_DISTANCE)
+	$Views/FrontRight.position = Vector2(_viewport_size.x + TEXTURE_DISTANCE, _viewport_size.y * -1 - TEXTURE_DISTANCE)
+	$Views/BackLeft.position = Vector2(_viewport_size.x * -1 - TEXTURE_DISTANCE, _viewport_size.y + TEXTURE_DISTANCE)
+	$Views/BackRight.position = Vector2(_viewport_size.x + TEXTURE_DISTANCE, _viewport_size.y + TEXTURE_DISTANCE)
+	
+
+# Set the current view
+#
+# ** Parameters **
+#
+# - value: The current view
+func _set_current_view(value: String):
+	emit_signal("view_changed", current_view, value)
+	current_view = value
+	EgoVenture.current_view = value
+	match current_view:
+		VIEW_FRONT: $Camera.position = Vector2(0, _viewport_size.y * -1 - TEXTURE_DISTANCE)
+		VIEW_RIGHT: $Camera.position = Vector2(_viewport_size.x + TEXTURE_DISTANCE, 0)
+		VIEW_BACK: $Camera.position = Vector2(0, _viewport_size.y + TEXTURE_DISTANCE)
+		VIEW_LEFT: $Camera.position = Vector2(_viewport_size.x * -1 - TEXTURE_DISTANCE, 0)
+		VIEW_FRONTLEFT: $Camera.position = Vector2(_viewport_size.x * -1 - TEXTURE_DISTANCE, _viewport_size.y * -1 - TEXTURE_DISTANCE)
+		VIEW_FRONTRIGHT: $Camera.position = Vector2(_viewport_size.x + TEXTURE_DISTANCE, _viewport_size.y * -1 - TEXTURE_DISTANCE)
+		VIEW_BACKLEFT: $Camera.position = Vector2(_viewport_size.x * -1 - TEXTURE_DISTANCE, _viewport_size.y + TEXTURE_DISTANCE)
+		VIEW_BACKRIGHT: $Camera.position = Vector2(_viewport_size.x + TEXTURE_DISTANCE, _viewport_size.y + TEXTURE_DISTANCE)
+
```

Comment:
Please add another new line here to comply with the [GDScript style guide](https://docs.godotengine.org/en/stable/getting_started/scripting/gdscript/gdscript_styleguide.html)